qutebrowser/utils/version.py

```diff
@@ -148,13 +148,18 @@ def _git_str_subprocess(gitpath):
     if not os.path.isdir(os.path.join(gitpath, ".git")):
         return None
     try:
-        cid = subprocess.check_output(
-            ['git', 'describe', '--tags', '--dirty', '--always'],
+        commit_hash = subprocess.check_output(
+            ['git',
+             'describe',
```

Comment:
nitpick: I think it's nicer to put those on a single line, as it's kind of a "sub-command"